frontend/src/constants/gameDefaults.ts

```diff
@@ -0,0 +1,4 @@
+import Technologies from './technologies';
+
+export const defaultTechnology = Technologies.HTML;
+export const defaultPlayersMeepleCount = 7;
```

Comment:
defaultTechnology and defaultPlayersMeepleCount can easly mix up with local variables - please name it as constant

Reply:
Done